syntax = "proto3";

enum ProtoSimpleEnum {
  Invalid = 0;
  ValueOne = 1;
  ValueTwo = 2;
}